docs/app/pages/components/components-sections/page-header/page-header/page-header.component.html

```diff
@@ -2,7 +2,47 @@
     [crumbs]="crumbs"
     [items]="items"
     [condensed]="condensed"
-    [iconMenus]="iconMenus">
+    [iconMenus]="iconMenus"
+    [secondaryNavigation]="secondaryNavigation"
```

Comment:
We decided not to add this to the open source doc site because it doesn't really make sense in that layout. The open source masthead is intrinsically hierarchical because every navigation item can be a dropdown. So I think you can leave this out and only worry about these options in the Micro Focus theme.